lib/muchos/existing.py

```diff
@@ -241,6 +241,10 @@ def perform(self, action):
             elif action == 'cancel_shutdown':
                 print("Cancelling automatic shutdown of {0} cluster".format(self.config.cluster_name))
             self.execute_playbook(action + ".yml")
+            # In case of Azure, also check and if necessary wipe the storage accounts
+            # It is necessary to do this after the common kill and wipe to ensure that processes are all killed prior to this
+            if  action == 'wipe' and self.config.get("general", "cluster_type") == 'azure':
```

Comment:
Would be nice to put azure specific code in azure.py if possible.  One possible way to do this is to create a `wipe()` method in existing.py that does nothing, override it in azure.py, and call `wipe()` here.  Alternatively, would it make sense for wipe.yml to call azure-wipe-adlsg2.yml in ansible instead of doing it from Pytyhon?

Reply:
Made the changes as per recommendation.

Reply:
We are unable to test wipe changes for EC2. Kindly please test wipe for EC2

Reply:
> Kindly please test wipe for EC2

Not sure I would have time to do that in the short term, not sure if anyone else could.  Looking at the diffs, it seems `wipe.yml` is called in the same way before and after this change.  So it seems likely that EC2 will work the same as before.